src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/compute/AzureComputePlugin.java

```diff
@@ -138,19 +147,31 @@ public ComputeInstance getInstance(ComputeOrder computeOrder, AzureUser azureUse
 
         AzureGetVirtualMachineRef azureGetVirtualMachineRef = this.azureVirtualMachineOperation
                 .doGetInstance(azureVirtualMachineId, azureUser);
-
+        
         return buildComputeInstance(azureGetVirtualMachineRef, azureUser);
     }
 
+    @VisibleForTesting
+    void updateInstanceAllocation(ComputeOrder computeOrder, VirtualMachineSize virtualMachineSize) {
+        synchronized (computeOrder) {
+            int vCPU = virtualMachineSize.numberOfCores();
+            int memory = virtualMachineSize.memoryInMB();
+            int disk = computeOrder.getDisk();
+            int instances = 1;
```

Comment:
Remove the magic number and add a constant

Reply:
When this branch merges with develop this variable will no longer exist.

Reply:
Ok, is done.